internal/controller/standalone_pgadmin/pod_test.go

```diff
@@ -49,15 +49,18 @@ containers:
   - bash
   - -ceu
   - --
-  - "monitor() {\nPGADMIN_DIR=/usr/local/lib/python3.11/site-packages/pgadmin4\n\necho
-    \"Running pgAdmin4 Setup\"\npython3 ${PGADMIN_DIR}/setup.py\n\necho \"Starting
-    pgAdmin4\"\nPGADMIN4_PIDFILE=/tmp/pgadmin4.pid\npgadmin4 &\necho $! > $PGADMIN4_PIDFILE\n\npython3
-    ${PGADMIN_DIR}/setup.py --load-servers /etc/pgadmin/conf.d/~postgres-operator/pgadmin-shared-clusters.json
-    --user admin@pgadmin.postgres-operator.svc --replace\n\nexec {fd}<> <(:)\nwhile
-    read -r -t 5 -u \"${fd}\" || true; do\n\tif [ \"${cluster_file}\" -nt \"/proc/self/fd/${fd}\"
-    ] && python3 ${PGADMIN_DIR}/setup.py --load-servers /etc/pgadmin/conf.d/~postgres-operator/pgadmin-shared-clusters.json
-    --user admin@pgadmin.postgres-operator.svc --replace\n\tthen\n\t\texec {fd}>&-
-    && exec {fd}<> <(:)\n\t\tstat --format='Loaded shared servers dated %y' \"${cluster_file}\"\n\tfi\n\tif
+  - "monitor() {\nPGADMIN_DIR=/usr/local/lib/python3.11/site-packages/pgadmin4\nAPP_RELEASE=$(cd
+    $PGADMIN_DIR && python3 -c \"import config; print(config.APP_RELEASE)\")\n\necho
+    \"Running pgAdmin4 Setup\"\nif [ $APP_RELEASE -eq 7 ]; then\npython3 ${PGADMIN_DIR}/setup.py\nelse\npython3
+    ${PGADMIN_DIR}/setup.py setup-db\nfi\n\necho \"Starting pgAdmin4\"\nPGADMIN4_PIDFILE=/tmp/pgadmin4.pid\npgadmin4
+    &\necho $! > $PGADMIN4_PIDFILE\n\nloadServerCommand() {\n\tif [ $APP_RELEASE -eq
```

Comment:
⛏️ I think this might be full of `\n` because of tabs in the value. If it matters, there are tests in other places that check for that:

https://github.com/CrunchyData/postgres-operator/blob/cf40aa9d8bd5be9e0ad835df3fd2b6ba59adaecf/internal/pgmonitor/exporter_test.go#L91-L96